### DISCUSSION TITLE
Replacing default glyphs with alternates

### DISCUSSION THREAD

**Discussion by Suckaboo** [fontforge/fontforge#4738] [Q&A]
Hey all, I'm pretty new to fontforge, and I've been searching as best I can for solutions, but no luck yet. The task at hand:

I was trying to find a way to replace the default glyphs in a font with the alternates already contained within that font. Specifically, the default numbers in [Google's free Raleway](https://fonts.google.com/specimen/Raleway?preview.text=J35%20VIGGEN&preview.text_type=custom&query=raleway) font are old style but I need to use the lining numerals. Not to mention the alts for G and W are also much nicer than the defaults.

Unfortunately I can't rely on alternate selection in layout software like adobe because this needs to be exported and embedded within our Unity project.

Any tips? Thanks in advance, and all the best to you!

**Comment by moyogo** [comment]
You can use https://pypi.org/project/opentype-feature-freezer/

**Reply by Suckaboo** [reply]
awesome tip thank you!
